app/views/communicart_mailer/new_attachment_email.html.haml

```diff
@@ -0,0 +1,9 @@
+%div
+  %p
+    Hello,
+    %br
+    An attachment has been added to #{@proposal.name} (#{@proposal.public_id}).
+
+%p
+  %strong
+    = link_to "View This Request", proposal_url(@proposal), {target: 'C2'}
```

Comment:
@phirefly Any strong feelings about what the email should say? We're currently sending this same thing to everyone. Thinking it should at least have the name of (and a link to) the attachment, but @annalee suggested that we might want to attach it to the email directly.